lib/util/markdown.ts

```diff
@@ -5,19 +5,32 @@ import { regEx } from './regex';
 // Generic replacements/link-breakers
 export function sanitizeMarkdown(markdown: string): string {
   let res = markdown;
+
+  // Replace spaces in Markdown links with %20.
+  const markdownLinks = res.match(/\(https?:\/\/[^)]*\)/g);
+
+  if (markdownLinks) {
+    markdownLinks.forEach((ml) => {
+      res = res.replace(ml, ml.replace(' ', '%20'));
```

Comment:
Won't this only replace the first only?

Reply:
But what about one link with two spaces?

Reply:
Fixed.